common/persistence/cassandraPersistence.go

```diff
@@ -1678,13 +1685,15 @@ func (d *cassandraPersistence) createTimerTasks(batch *gocql.Batch, timerTasks [
 
 	for _, task := range timerTasks {
 		var eventID int64
+		var attempt int64
 
 		timeoutType := 0
 
 		switch task.GetType() {
 		case TaskTypeDecisionTimeout:
 			eventID = task.(*DecisionTimeoutTask).EventID
 			timeoutType = task.(*DecisionTimeoutTask).TimeoutType
+			attempt = task.(*DecisionTimeoutTask).ScheduleAttempt
```

Comment:
Wouldn't it be cleaner to introduce a local variable to avoid all these casts?

Reply:
Done.